string skill_id
---
int32 request_result_id
string request_result_label
